### PR TITLE
Josecc/responsive cards

### LINKED ISSUE #81: Responsive for cards SCSS
Add breakpoints to make the cards responsive.

### PR DESCRIPTION
**Pull request by Josecc**
This will resolve #81 by adding breakpoints and will also allow the user to view the website on their phone. Next steps are to fix navigation flow because its hard to type urls on the phone.

### CODE REVIEW

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -1,5 +1,16 @@
 <nav class="navbar navbar-inverse navbar-static-top">
   <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <% if session[:organization_name] %>
```
Comment (nolds9) [inline app/views/layouts/_navbar.html.erb:10]: is this implemented in a subsequent commit?

**Comment by Josecc** [comment]
No this works now, its just the hamburger bar for phones (the menu bar) and the title of the organization

File: app/views/layouts/_navbar.html.erb (hunk 1 above)
Comment (jaydorsey) [inline app/views/layouts/_navbar.html.erb:10]: It was already in this branch. The icon doesn't show up until you shrink the screen size to a certain (smaller) point

Comment (Josecc) [inline app/views/layouts/_navbar.html.erb:10]: No this works now, its just the hamburger bar for phones (the menu bar) and the title of the organization

**Pull request merged**